Gemfile

```diff
@@ -47,8 +47,13 @@ gem "tzinfo-data"
 gem "govuk-components", ">= 1.0.0"
 gem "govuk_design_system_formbuilder", "~> 2.1", ">= 2.1.5"
 gem "govuk_publishing_components", ">= 23.11.0"
+gem "httpclient"
```

Comment:
Just curious is this what's mostly used in dfe? Most organizations tend to stick to one http client and probably there's one in dfe that's widely used. It could be this one though.

Reply:
I got that out of `claim-additional-payments-for-teaching`.

DfE clearly doesn't stick to one - I saw `http` gem in `apply-for-teacher-training`, `httparty` in `teaching-vacancies`, and `httpclient` in `claim-additional-payments-for-teaching`.

The convenience of using this one is that copy-pasting it alongside a bunch of other things from `claim-additional-payments-for-teaching` 'just works'.